### ISSUE TITLE
Training instructions from README.md not working for me

### ISSUE THREAD

**Issue by davidmartinrius** [lifeiteng/vall-e#104]
Hello,

I am working with an Ubuntu 22, a NVIDIA RTX 3080, 64GB RAM

I followed the steps of the DEMO in the README.md to train a model of LibriTTS.

![image](https://user-images.githubusercontent.com/16558194/233652177-10a247ae-88a3-4f38-9ba3-2971eeaaac31.png)

The result after the inference is wrong. It sounds like a weird noise. I attached the wav inside a zip because github does not allow to upload a wav.

[0.zip](https://github.com/lifeiteng/vall-e/files/11296265/0.zip)

Please, can you help me to understand what am I doing wrong?

Ask me for any information you need to analyze, I will provide it.

Thank you,

David Martin Rius

**Comment by lifeiteng** [comment]
@davidmartinrius The problem is `--max-duration to 20` which means the batch_size is in `[1, 9]`.

try to train the model on 3090/4090 or A100.

**Comment by davidmartinrius** [comment]
@lifeiteng thanks for your response. Your response is not clear for me. Maybe the batch size is lower because of the vram, and it means that needs more iterations, but it should not affect the performance. I'm sorry but your response is not useful for me. It should be able to train it in almost any Nvidia RTX > 3000 series GPU...

Please, if you think max duration is the problem, can you explain how to adapt it to a 10GB GPU?...

**Comment by lifeiteng** [comment]
@davidmartinrius 
Small batch_size will not converge to a good local optimal point. It's common sense in DeepLearning.

**Comment by davidmartinrius** [comment]
I agree with you at this point. I understand that when the batch size is too small, the gradients computed from the batch may not be representative of the overall structure of the dataset, leading to unstable and slow convergence during training.

Do you think is possible to make it work adjusting the gradient accumulation, the learning rate, batch normalization or even adding more layers? Actually I don't know the whole project, maybe you could valuate it.

If there is a way to optimize it I would like to try it. I know it means more training hours and more development. 

Thank you!

David Martin Rius
